packages/ui-kit-theme/src/breakpoints.ts

```diff
@@ -0,0 +1,23 @@
+declare module '@mui/material/styles' {
+  interface BreakpointOverrides {
+    xxs: true;
+    xs: true;
+    sm: true;
+    md: true;
+    lg: true;
+    xl: true;
+  }
+}
+
+export default {
+  breakpoints: {
```

Comment:
This block could be exported instead of the containing block to keep the pattern the same as the other theme property files.